pom.xml

```diff
@@ -31,7 +31,7 @@
 
     <properties>
         <inceptionYear>2022</inceptionYear>
-        <opentelemetry.spec.version>1.13.0</opentelemetry.spec.version>
+        <opentelemetry.spec.version>1.14.0</opentelemetry.spec.version>
         <opentelemetry.java.version>1.18.0</opentelemetry.java.version>
```

Comment:
```suggestion
        <opentelemetry.java.version>1.19.0</opentelemetry.java.version>
```

Reply:
I will update to the new Java version - this was not possible before as the opentelemetry-java-instrumentation part was missing - but now it's released.
Originally, @brunobat and me split up the changes (handeled in 2 issues) - but I can add that now in this PR too, if you like.